manage_certs.py

```diff
@@ -45,6 +45,7 @@ def parse_command_line(args):
     parser.add_argument("--consul-token")
     parser.add_argument("--deploy-hook", default='/usr/local/sbin/deploy_cert.sh')
     parser.add_argument("--dns-delay", type=int, default=120)
+    parser.add_argument("--email-failure-alert", action='store_true')
```

Comment:
@ahmedbilal, this has to be updated to accept an email address - see [this comment|https://github.com/open-craft/cert-manager/pull/15/files#r467742013] and the `contact email address` could be the default value.